lib/ffaker/number.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module FFaker
+  module Number
+    extend ModuleUtils
+    extend self
+
+    def number(digits: 1)
+      FFaker.numerify(generate_pattern(digits)).to_i
+    end
+
+    def decimal(whole_part_digits: 1, fractional_part_digits: 1)
+      whole_part_pattern = generate_pattern(whole_part_digits)
+      fractional_part_pattern = generate_pattern(fractional_part_digits).reverse
+      FFaker.numerify("#{whole_part_pattern}.#{fractional_part_pattern}").to_f
+    end
+
+    private
+
+    def generate_pattern(digits)
+      raise ArgumentError.new('Digits cannot be less than 1') if digits < 1
+
+      non_zero_numbers = (1..9).to_a
+      "#{fetch_sample(non_zero_numbers)}#{"#" * (digits - 1)}"
```

Comment:
Looks like fetch_sample supports ranges: https://github.com/ffaker/ffaker/blob/b661ddb32027c2abac66a5162b1198e3ea553c3e/lib/ffaker/utils/random_utils.rb#L23

```suggestion
      "#{fetch_sample(1..9)}#{"#" * (digits - 1)}"
```